index.js

```diff
@@ -205,6 +205,8 @@ async function installBundler(bundlerVersionInput, lockFile, platform, rubyPrefi
     throw new Error(`Cannot parse bundler input: ${bundlerVersion}`)
   }
 
+  core.exportVariable('BUNDLER_VERSION', bundlerVersion)
```

Comment:
This looks much much simpler :)

I'm not sure we should export `BUNDLER_VERSION` for the whole workflow, but OTOH it's probably clearer that way and it should not hurt.

Can you move this line at the end of this function?
Notably it needs to consider `bundlerVersion` being assigned below.

Reply:
ouch, missed that it's assigned below. Fixed.